agents/simple_agent.py

```diff
@@ -34,29 +40,48 @@ def act(self, observation):
     """Act based on an observation."""
     if observation['current_player_offset'] != 0:
       return None
+   
+    '''
+    print ("Config:")
+    print(self.config)
+   
+    print("Act:")
+    if observation is not None:
+      print("OBSERVATION ================================================")
+      print(observation)
+      print("===========================================================")
+    '''
+ 
 
     # Check if there are any pending hints and play the card corresponding to
     # the hint.
-    for card_index, hint in enumerate(observation['card_knowledge'][0]):
```

Comment:
Changing the agent's algorithm to demonstrate rainbow cards:

Previous algorithm:
   - If there is a hint, then play it,
   - Otherwise, hint for any playable cards

Now: hint all rainbow cards xD